### PR TITLE
add `getTables` method for 4.3.x branch

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

File: scalikejdbc-core/src/main/scala/scalikejdbc/metadata/Table.scala [hunk 1]
```diff
@@ -19,6 +19,9 @@ case class Table(
   indices: List[Index] = Nil
 ) {
 
+  // could not use case class field for binary compatibility
+  def catalog: String = null
```
Comment (xuwei-k) [inline scalikejdbc-core/src/main/scala/scalikejdbc/metadata/Table.scala:23]: 😢

**Comment by seratch** [comment]
We’ve already released v4.3.0 a few weeks ago. Can you submit this for 4.3.x branch instead?

**Comment by xuwei-k** [comment]
> Can you submit this for 4.3.x branch instead?

done 🙏

**Review by seratch — APPROVED** [review]
Thank you!

**Pull request merged**